ci/cron/daily-compat.yml

```diff
@@ -258,4 +264,20 @@ jobs:
         displayName: 'Blackduck Scan'
         env:
           BLACKDUCK_HUBDETECT_TOKEN: $(BLACKDUCK_HUBDETECT_TOKEN)
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: |
+          set -euo pipefail
+          eval "$(./dev-env/bin/dade-assist)"
+          source $(bash_lib)
+
+          tr -d '\015' <*_Black_Duck_Notices_Report.txt | grep -v dach-ny_daml-on-corda >NOTICES
```

Comment:
Why is daml on corda included here at all?

Reply:
I've wondered about that too. This line is coming directly from NOTICES.md in #8161. Assuming I get to a working state I'll try removing the `grep -v` bit and see how that goes.

@dasormeter any comment on what this `grep -v` command does?

Reply:
the line in question in the auto-generated file looks like
```[digital-asset_daml : master]```